dev/tools/gen_keycodes/README.md

```diff
@@ -58,109 +58,62 @@ the API.
-  appear in the app as 0x100 0007 00E8, and the app could test against that
-  code. Yes, this also means that once they recompile with a version of
-  Flutter that supports this new HID code, apps looking for this code will
-  break. This situation is only meant to provide a fallback ability for apps
-  to handle esoteric codes that their version of Flutter doesn’t support yet.
-  The prefix for this code is the platform prefix from the previous sections,
-  plus 0x100.
-
-- **0x200 0000 0000 - 0x2FF FFFF FFFF**: For pseudo-keys which represent
-  combinations of other keys, and conceptual keys which don't have a physical
-  representation. This is where things like key synonyms are defined (e.g.
-  "shiftLeft" is a synonym for "shift": the "shift" key is a pseudo-key
-  representing either the left or right shift key).
-
-**This is intended to get us out of the business of defining key codes where
-possible.** We still have to have mapping tables, but at least the actual minting
-of codes is deferred to other organizations to a large extent. Coming up with a
-code is a mechanical process consisting of just picking the lowest number code
-possible that matches the semantic meaning of the key according to the
-definitions above.
-
-Here are some examples:
-
-For example, on a French keyboard layout, pressing CAPS LOCK then pressing
-SHIFT + Y would generate the following sequence:
-
-DOWN, code 0x0100070039. (CAPS LOCK DOWN)<br>
-UP, code 0x0100070039. (CAPS LOCK UP)<br>
-DOWN, code 0x01000700E1 (LEFT SHIFT DOWN)<br>
-DOWN, code 0x0000000059, string U+00059 (Y DOWN)<br>
-UP, code 0x0000000059 (Y UP)<br>
-UP, code 0x01000700E1 (LEFT SHIFT UP)<br>
-
-Here's another example. On a German keyboard layout, you press ^e (the ^ key is
-at the top left of the keyboard and is a dead key) to produce an “ê”:
-
-DOWN, code 0x0000000302 (CIRCUMFLEX DOWN) It produces no string, because it's a dead
-key. The key code is for "Combining circumflex accent U+0302" in Unicode.<br>
-UP, code 0x0000000302 (CIRCUMFLEX UP)<br>
-DOWN, code 0x0000000065, string U+000EA (Unicode for ê‬) (E DOWN).<br>
-UP, code 0x0000000065. (E UP).<br>
-
-It is an important point that even though we’re representing many keys with USB
-HID codes, these are not necessarily the same HID codes produced by the hardware
-and presented to the driver, since on most platforms we have to map the platform
-representation back to an HID code because we don’t have access to the original
-HID code. USB HID is simply a conveniently well-defined standard that includes
-many of the keys we would want.
+  | 0x11 | Android  |
+  | 0x12 | Fuchsia  |
+  | 0x13 | iOS      |
+  | 0x14 | macOS    |
+  | 0x15 | Gtk      |
+  | 0x16 | Windows  |
+  | 0x17 | Web      |
+  | 0x18 | GLFW     |
+
+  Platform planes store keys that are private to the Flutter embedding of this
+  platform.  This most likely means that these keys are unrecognizable and are
```

Comment:
I've changed it to the following:
```

  Platform planes store keys that are private to the Flutter embedding of this
  platform. This most likely means that these keys have not been officially
  recognized by Flutter.

  The value scheme within a platform plane is decided by the platform,
  typically using the field from the platform's native key event that
  represents the key's logical effect (such as `keycode`, `virtual key`, etc).

  In time, keys that originally belong to a platform plane might be added to
  Flutter, especially if a key is found shared by multiple platforms. The values
  of these keys will be changed to a new value within the Flutter plane, and
  all platforms managed by Flutter will start to send the new value, making it
  a breaking change. Therefore, when handling an unrecognized key on a platform
  managed by Flutter, it is recommended to file a new issue to add this value
  to `keyboard_key.dart` instead of using the platform-plane value. However,
  for a custom platform (see below), since the platfrom author has full control
  over key mapping, such change will not cause breakage and it is recommended
  to use the platform-plane value to avoid adding platform-exclusive values
  to the framework.
```